api_tests/nodes/views/test_node_alternative_citations.py

```diff
@@ -0,0 +1,1098 @@
+from nose.tools import *  # flake8: noqa
+
+from website.util import permissions
+
+from api.base.settings.defaults import API_BASE
+
+from tests.base import ApiTestCase
+
+from tests.factories import (
+    ProjectFactory,
+    RegistrationFactory,
+    AuthUserFactory,
+    AlternativeCitationFactory
+)
+
+class TestCreateAlternativeCitations(ApiTestCase):
```

Comment:
Can you assert that you're getting the expected JSON response data in all of your tests as well? For example, if you update the citation text assert that `res.json['data']['attributes']['text']` is equal to `payload['data']['attributes']['text']`.